pkg/log/file_test.go

```diff
@@ -0,0 +1,40 @@
+package log
+
+import (
+	"testing"
+
+	. "github.com/smartystreets/goconvey/convey"
+)
+
+func (w *FileLogWriter) WriteLine(line string) error {
+	n, err := w.mw.Write([]byte(line))
+	if err != nil {
+		return err
+	}
+	w.docheck(n)
+	return nil
+}
+
+func TestLogFile(t *testing.T) {
+
+	Convey("When logging to file", t, func() {
+		fileLogWrite := NewFileWriter()
+		So(fileLogWrite, ShouldNotBeNil)
+
+		fileLogWrite.Filename = "../../data/log/grafana.log"
```

Comment:
using the standard logfile will fail when running the test locally if my log file have content. Just use a local file in this directory. The path is not important in this case.